### PR TITLE
additionally activate extension when files are saved

### PR DESCRIPTION
**Pull request by discentem**
Tested locally with process described at https://code.visualstudio.com/api/get-started/your-first-extension:

The extension compiles just fine, opens `[Extension Development Host]` in a new window, and I observe syntax highlighting works on file save.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "shebang-language-associator",
   "displayName": "Shebang Language Associator",
   "description": "A tiny extension to associate shebang lines with languages",
-  "version": "1.2.0",
+  "version": "1.2.1",
```
Comment (davidhewitt) [inline package.json]: Nit: I think really this is a new feature so let's bump the minor version.

```suggestion
  "version": "1.3.0",
```

**Review by davidhewitt — COMMENTED** [review]
Thanks for suggesting and implementing this! I went back and forth on having an option, in the end I think it's unlikely to be a major resource concern (compared to the other operations which often happen on save like formatting).

If you can also add a new section to the CHANGELOG, then I'll be happy to give this a quick test later & release. Thanks 👍

**Comment by discentem** [comment]
> Thanks for suggesting and implementing this! I went back and forth on having an option, in the end I think it's unlikely to be a major resource concern (compared to the other operations which often happen on save like formatting).

👍 

> If you can also add a new section to the CHANGELOG, then I'll be happy to give this a quick test later & release. Thanks 👍

Done!

**Review by davidhewitt — APPROVED** [review]
Thanks, I will try to find a chance to verify and release later today!

**Pull request merged**